website/src/pages/api/valid_labels.ts

```diff
@@ -0,0 +1,23 @@
+import { getToken } from "next-auth/jwt";
+import { oasstApiClient } from "src/lib/oasst_api_client";
+
+/**
+ * TODO
```

Comment:
todo what?

Reply:
I see you did the review carefully.  TODO was to write the handler comments.  done :)